src/components/HyperlinkingWrapper.js

```diff
@@ -129,6 +144,8 @@ export default class HyperlinkingWrapper extends Component {
 
     renderTooltip() {
         const { current, isLinkInvalid, currentSelection } = this.state;
+        const format = this.quill.getFormat();
+        const linkValue = (typeof format.link === 'string') ? format.link : null;
         const isEdit = current === HYPERLINKING_STATE.EDIT;
```

Comment:
I think for editing we should set `this.state.current = 'HYPERLINKING_STATE.EDIT';`, so we show InputTooltip at once.